centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl

```diff
@@ -5,7 +5,6 @@
     <title-short>CRCAO</title-short>
     <id>http://www.zotero.org/styles/centre-de-recherche-sur-les-civilisations-de-l-asie-orientale</id>
     <link href="http://www.zotero.org/styles/centre-de-recherche-sur-les-civilisations-de-l-asie-orientale" rel="self"/>
-    <link href="https://github.com/citation-style-language/styles/pull/1385#issuecomment-73234582" rel="documentation"/>
```

Comment:
No there is no reason. It's just a mistake, but I hope it's now OK.

Thank you very much !
Philippe Pons

Le 19/02/2016 15:48, Rintze M. Zelle a �crit :
> <link href="https://github.com/citation-style-language/styles/pull/1385#issuecomment-73234582" rel="documentation"/>